portfolio/src/main/webapp/index.html

```diff
@@ -8,9 +8,33 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+			<header>
+				<h1>Abraham Haros's Portfolio</h1>
+			</header>
+			<article>
+      <h2>About me.</h2>
+				<p>Since childhood I have been passionate about software development, 
+					in middle school I was  interested on learning how a computers works, 
+					so I decided to enroll in an introduction to web development course, 
+					that was a very satisfying experience and made me decide for more knowledge 
+					in the field, hence in high school I enter to a software development program 
+					where I learned many things, such as Object Oriented Programming, mobile app 
+					development and database management.
+				<p>For me, applying the knowledge to real life problems is the best way to 
+					learn about something, hence why I started to make my first personal projects 
+					on high school. I have worked in solutions for my classes, for example, 
+					implementing c++ functions to evaluate physics functions.
```

Comment:
```suggestion
					implementing C++ functions to evaluate physics functions.</p>
```
Same thing here :) (Also capitalize "C++")